tutorials/FF/pipe-pipe/sonicLiquidFoam-sonicLiquidFoam/precice-config.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+
+<precice-configuration>
+
+    <log>
+        <sink filter="%Severity% > debug and %Rank% = 0" format="---[precice] %ColorizedSeverity% %Message%" enabled="true"/>
+    </log>
+
+    <solver-interface dimensions="3">
+
+    <data:vector name="VelocityGradient"/>
+    <data:scalar name="Pressure"/>
+    <data:vector name="Velocity"/>
+    <data:scalar name="PressureGradient"/>
+
+
+    <mesh name="Fluid1-Mesh">
+        <use-data name="VelocityGradient"/>
+        <use-data name="Pressure"/>
+        <use-data name="Velocity"/>
+        <use-data name="PressureGradient"/>        
+    </mesh>
+
+    <mesh name="Fluid2-Mesh">
+        <use-data name="VelocityGradient"/>
+        <use-data name="Pressure"/>
+        <use-data name="Velocity"/>
+        <use-data name="PressureGradient"/>
+    </mesh>
+
+    <participant name="Fluid1">
+        <use-mesh name="Fluid1-Mesh" provide="yes"/>
+        <use-mesh name="Fluid2-Mesh" from="Fluid2"/>
+        <write-data name="Velocity" mesh="Fluid1-Mesh"/>
+        <write-data name="PressureGradient" mesh="Fluid1-Mesh"/>
+        <read-data name="Pressure" mesh="Fluid1-Mesh"/>
+        <read-data name="VelocityGradient" mesh="Fluid1-Mesh"/>
+        <mapping:nearest-neighbor direction="read" from="Fluid2-Mesh" to="Fluid1-Mesh" constraint="consistent" />
+    </participant>
+
+    <participant name="Fluid2">
+        <use-mesh name="Fluid2-Mesh" provide="yes"/>
+        <use-mesh name="Fluid1-Mesh" from="Fluid1"/>
+        <read-data name="Velocity" mesh="Fluid2-Mesh"/>
+        <read-data name="PressureGradient" mesh="Fluid2-Mesh"/>
+        <write-data name="Pressure" mesh="Fluid2-Mesh"/>
+        <write-data name="VelocityGradient" mesh="Fluid2-Mesh"/>
+        <mapping:nearest-neighbor direction="read" from="Fluid1-Mesh" to="Fluid2-Mesh" constraint="consistent" />
+    </participant>
+
+    <m2n:sockets from="Fluid1" to="Fluid2"/>
+
+    <coupling-scheme:serial-implicit>
+        <time-window-size value="0.01"/>
+        <max-time value="1.0"/>
+        <participants first="Fluid1" second="Fluid2"/>
+        <exchange data="Velocity" mesh="Fluid1-Mesh" from="Fluid1" to="Fluid2"/>
+        <exchange data="PressureGradient" mesh="Fluid1-Mesh" from="Fluid1" to="Fluid2"/>
+        <exchange data="Pressure" mesh="Fluid2-Mesh" from="Fluid2" to="Fluid1"/>
+        <!--<exchange data="VelocityGradient" mesh="Fluid2-Mesh" from="Fluid2" to="Fluid1"/>-->
```

Comment:
This combination diverges very early in the simulation.

Reply:
The problem was the too weak convergence of OpenFOAM combined with the too strict convergence measures of preCICE.

Reply:
Oh, I was working on this already, thanks!